db/seeds.rb

```diff
@@ -1,33 +1,40 @@
-Rails.logger.info "1/9 Creating Organizations"
+Rails.logger.info "1/10 Creating Organizations"
 
-3.times do
+40.times do
```

Comment:
What about using a multiplicator as a parameter (in the case of `db:seed` it'd probably be an env variable)? 

Let's say the default seeds keeps things as they are and one can say "generate 2 times the default amount of data" or 10 times that.

Reply:
I increased the number enough for the purpose of pagination, and I think we should ensure we have enough data by default so when people go to Q&A the app has what they need without having to look or fiddle with settings.

In honest, this is such a small increase in seed data where I have web-apps which load 10K records within a minute or less. 

The seed data is slow in the respo because of architectual choices such as using `acts_as_taggable`. I can certaintly suggest in future tickets to improve these performance issues so we can much more seed data.